### PR TITLE
Add edge analytic pages

### PR DESCRIPTION
**Pull request by pgerv12**

### CODE REVIEW

File: docs/edgeanalytics/kafka-options.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+## Event Streams in IBM Cloud
+
+[IBM Event Streams](https://www.ibm.com/cloud/event-streams) builds on top of open source Apache Kafka to offer enterprise-grade event streaming capabilities.
+
+### Provisioning Event Streams
+
+1. Log in to [IBM Cloud](https://cloud.ibm.com) or create an account if you do not have one.
+1. Visit [Event Streams](https://cloud.ibm.com/catalog/services/event-streams) in the catalog.
+1. Select a region (e.g. Dallas, Frankfurt)
+1. Select a plan (e.g. Lite)
```
Comment (natashadsilva) [inline docs/edgeanalytics/kafka-options.md:131]: Can you please add a comment that the Lite plan only allows one topic which might not work with some of the samples?

File: docs/edgeanalytics/kafka-options.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+* **SPL applications**: see [`streamsx.messagehub` samples](https://ibmstreams.github.io/streamsx.messagehub/docs/user/overview/#samples) for connection configuration options.
+
+***
+
+## Vanilla Apache Kafka
+
+Apache Kafka can be deployed on bare metal/VM systems as well as Kubernetes and OpenShift.
+
+Because users should already have access to a Kubernetes-like environment, the following install section will cover Helm charts and Operators to deploy Kafka servers.
+
```
Comment (natashadsilva) [inline docs/edgeanalytics/kafka-options.md:165]: Just to make sure I understand, if for some reason they're not using k8s and they have Kafka installed anywhere else, that system must be accessible from the CPD cluster, right?

File: docs/edgeanalytics/index.markdown [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+tag: edge
+next:
+  file: edgeanalytics/kafka-options
+  title: Apache Kafka options for edge applications
+---
+
+IBM® Streams can be used to develop and build applications for edge analyics using IBM Edge Analytics Beta on Cloud Pak for Data.
+
+For more information about Edge Analytics, see the [IBM Edge Analytics Beta on Cloud Pak for Data](https:// https://www.ibm.com/support/knowledgecenter/SSQNUZ_3.0.1/svc-welcome/edge.html) documentation.
+
```
Comment (natashadsilva) [inline docs/edgeanalytics/index.markdown:17]: Please add a link to samples once they are available.

File: docs/edgeanalytics/kafka-options.md (hunk 2 above)
Comment (pgerv12) [reply docs/edgeanalytics/kafka-options.md:165]: The Kafka system does not need to necessarily be accessable from the CPD cluster. If customers want the data from that Kakfa system to be used in other CPD services, then yes. I'll add a stub about this.

**Pull request merged**